src/services/repo/useRepoOverview.tsx

```diff
@@ -135,5 +144,6 @@ export function useRepoOverview({
         }
       })
     },
+    enabled,
```

Comment:
What is this for?

Reply:
From [TanStack Query Docs](https://tanstack.com/query/v4/docs/framework/react/reference/useQuery):

![Screenshot 2024-02-26 at 14 06 34](https://github.com/codecov/gazebo/assets/105234307/3ead3e09-f6ad-49ac-bdd1-e40bc3840fcf)

Reply:
It allows us to stop a query (and skip a network request) in this instance I've added it in, because in the hook where I'm calling it I use it to grab the default branch if there is no passed branch. But if there is a passed branch I don't need to grab the default branch so I can just skip the request altogether.

Reply:
Ahhh okay that makes sense! Thanks @nicholas-codecov